armi/physics/neutronics/settings.py

```diff
@@ -342,251 +343,170 @@ def defineSettings():
     return settings
 
 
-# OLD STYLE settings rules from settingsRules.py. Prefer validators moving forward.
-
-
-@include_as_rule("genXS")
-def newGenXSOptions(_cs, name, value):
-    """
-    Set new values of 'genXS' setting based on previous setting values.
-
-    Arguments
-    ---------
-    cs : setting object
-        ARMi object containing the default and user-specified settings.
-
-    name : str
-        Setting name to be modified by this rule.
-
-    value : str
-        Value of the setting identified by name.
-
-    Returns
-    -------
-    dict : dict
-        Updated setting name and or value.
-    """
-    if value == "True":
-        newValue = NEUTRON
-    elif value == "False":
-        newValue = ""
-    else:
-        newValue = value
-
-    if value != newValue:
-        runLog.info("The `genXS` setting has been set to {}.".format(newValue))
-    return {name: newValue}
-
-
-@include_as_rule("existingFIXSRC")
-def newFixedSourceOption(cs, _name, value):
```

Comment:
Did we miss this migration function when you revamped this file? Or did you mean to remove it without replacement?

Reply:
As far as I can tell, it is not used anywhere downstream.  If I am mistaken, by all means, holler.  But I couldn't find any references to it anywhere.